### ISSUE TITLE
Layout needs to be optimised

### ISSUE THREAD

**Issue by adriantache** [adriantache/Bowling-Scoring#1]
Layout is not properly responsive, and does not completely work on tiny screens.

**Comment by Safirah** [comment]
Can I start working first on this, as discussed on pull request #7 ?

**Comment by adriantache** [comment]
Absolutely, but please use a separate `layout-***.xml` file in order to preserve default layout.

**Comment by Safirah** [comment]
@adriantache I'm thinking in making it more responsive, so probably will not change it's appearance in bigger screens. But could you send me a screenshoot of how the xml gets displayed on your phone, so I can use it as a reference?

**Comment by adriantache** [comment]
There is one in the `readme.md` in the root folder. However, the app may be designed for res higher than `xxxhdpi` (my phone is `xxxhdpi` but my `Display size` is set to `small`, leading to a higher dpi than `xxxhdpi`).
